### PR TITLE
feat: add uat network

### PR DESCRIPTION
**Pull request by pkucherenk0**

### CODE REVIEW

**Comment by pkucherenk0** [comment]
1. removed networks from assets that are not used by the file
2. add uat file to be used with https://ethnodescan.v4.uat.opendax.app/

File: networks/canarynet-uat/peers.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            {
+              "name": "prime",
+              "margin": [100, 1000],
+              "safety": "3.5"
+            }
+          ]
+    },
+    {
+        "url": "https://api.kayen.io/",
+        "name": "kayen.eth",
```
Comment (philanton) [inline networks/canarynet-uat/peers.json:21]:
```suggestion
        "name": "kayen.ynt",
```

File: networks/canarynet-uat/peers.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            },
+            {
+              "name": "prime",
+              "margin": [100, 1000],
+              "safety": "3.5"
+            }
+          ]
+    },
+    {
+        "url": "https://api.kayen.io/",
```
Comment (philanton) [inline networks/canarynet-uat/peers.json:20]: This we will know only after few days, so even next 2 comments can omit for now

File: networks/canarynet-uat/mapping.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "BTC": ["BTC","WBTC"],
+    "ETH": ["ETH","WETH"],
+    "USD": ["USDT"]
```
[the remark this answers is not in the document]
Comment (pkucherenk0) [reply networks/canarynet-uat/mapping.json:4]: We can, but should not right now. The price index will become much more complicated if we need to look for the price of WBTC/USDT and WBTC/USDC markets.

File: networks/canarynet-uat/peers.json (hunk 2 above)
Comment (pkucherenk0) [reply networks/canarynet-uat/peers.json:20]: https://kayen.v4.uat.opendax.app/ maybe?

File: networks/canarynet-uat/assets.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (mod) [inline networks/canarynet-uat/assets.json:1]: rename to stagenet
and we might also need localnet (matching the docker-compose env.)

File: networks/canarynet-uat/peers.json (hunk 2 above)
Comment (philanton) [reply networks/canarynet-uat/peers.json:20]: http://clearport.v4.uat.opendax.app:30051

**Review by philanton — CHANGES_REQUESTED** [review]

**Pull request merged**